src/platforms/dotnet/common/troubleshooting.mdx

```diff
@@ -4,4 +4,11 @@ sidebar_order: 100
 description: "Learn more about how to troubleshoot common issues with the .NET SDK. "
 ---
 
+## User IP address in Sentry shows up as the server IP
+
+If you're using a proxy server that relies on `X-Forwarded-For`, you might need to [configure ASP.NET Core
+so that it's aware of it](https://docs.microsoft.com/aspnet/core/host-and-deploy/proxy-load-balancer).
```

Comment:
"so that it's aware of it" is a bit awkward since the two its aren't the same thing. Suggest: 

```suggestion
so that it's aware of the request header](https://docs.microsoft.com/aspnet/core/host-and-deploy/proxy-load-balancer).
```